### PR TITLE
Feature/diagnostics

### PR DESCRIPTION
**Pull request by erlemantos**

### CODE REVIEW

File: repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/diagnostics/DiagnosticsController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/rest/{tenant}/diagnostics")
+public class DiagnosticsController {
+
+	@Autowired
+	private JcrModelRepository modelRepo;
+	
+	@RequestMapping(method = RequestMethod.GET)
```
Comment (aedelmann) [inline repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/diagnostics/DiagnosticsController.java]: The diagnose endpoint is not secured by a specific role ?

Comment (erlemantos) [reply repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/diagnostics/DiagnosticsController.java]: Yeah, I forgot that one.

**Comment by aedelmann** [comment]
Hi @erlemantos 
I created an issue for this diagnostics functionality: #1166 
Can you check if the description fits to what is implemented ?

**Comment by erlemantos** [comment]
It should. Compliance to Vorto meta model is ensured by model validation checks. There's a check for references to, and references from. There's a metadata test if metadata conforms to model.

**Comment by aedelmann** [comment]
Awesome. I add this additional info in the issue for tracking.

**Pull request merged**